File: SqftToSqM/Convert.java
/*
    Author : Pakshal Jain
    Date : 30/10/2022
    Program : To Convert Entered SqFt To SqM
*/

//Required Libraries
import java.lang.*;
import java.util.*;

/////////////////////////////////////////////////////////////////////////////

//Class Declaration
class SqftToSqM
{
    //Characterstics of Class
    float Value = 0.0f;

    //Constructor of Class
    SqftToSqM(float value)
    {
        this.Value = value;
    }

    //Behaviour of Class
    double convert()
    {
        double ans = 0.0f;

        ans = this.Value * 0.0929;

        return ans;
    }
}

/////////////////////////////////////////////////////////////////////////////////////////////////

//Main Class
class Convert
{
    public static void main(String args[])
    {
        float No = 0;
        double dret = 0.0f;

        //Cretaing Object of Scanner Class To Take Input From User
        Scanner sobj = new Scanner(System.in);

        System.out.println("Jay Ganesh...");

        System.out.println("Enter Square Feet To Convert in meter...");
        
        No = sobj.nextFloat();

        //Creating object of Sqft Class To
        SqftToSqM obj = new SqftToSqM(No);
        
        //Call to behaviour of class
        dret = obj.convert();

        System.out.println("Sqft Converted To SqM is : "+dret);
    }
}

File: SqftToSqM/Convert.py
"""
    Author : Pakshal Jain
    Date : 30/10/2022
    Program : To Convert Entered Sqft To SqM
"""
#####################################################################################################

#Function TO Convert Sqft To SqM
def Convert(No) :
    ans = 0.0;

    ans = No * 0.0929;

    return ans;

######################################################################################################

#Main Function
def main() :
    No = 0;
    ret = 0.0;

    print("Jay Ganesh...");
    
    print("Enter Sqft to Convert into SqM");
    No = int(input());

    #Call TO Conver Function
    ret = Convert(No);

    print("Convert Sqft To SqM is : ",ret);

#Call Back Method To Call Main Function
if __name__ == "__main__" :
    #Call To Main Function
    main();